HaplotypeMap/BuildHaplotypeMap.wdl

```diff
@@ -0,0 +1,167 @@
+# BuildHaplotypeMap
+#
+# This workflow creates a haplotype map file containing a set of low LD sites for fingerprinting
+#
+# Inputs:
+#
+# 1. Genotyped multisample GVCF
+# 2. Header file. Standard SAM header (should match header of files to be fingerprinted)
+# 3. Pruning parameters, see https://www.cog-genomics.org/plink/2.0/ld for more info. 
+#    In general, higher values for prune window and prune slide result in more pruning, while lower values for prune cutoff and maf result in less pruning
+#  
+#   
+#
+
+task vcftools {
+    String intermediates_prefix
+    File input_vcf
+    
+    command <<<
+  	vcftools --vcf ${input_vcf} --max-missing-count 0 --remove-indels --recode --recode-INFO-all --out ${intermediates_prefix}
+    >>>
+	
+    runtime {
+    disks: "local-disk 1000 HDD"
+    memory: "3500 MB"
+    docker: "jeltje/vcftools"
+  }
+
+	output {
+    File recode_vcf = "${intermediates_prefix}.recode.vcf"
+  }
+}
+
+task plink {
+  File input_vcf
+  String intermediates_prefix
+  String output_prefix
+  Int prune_window
```

Comment:
these pruning related values should be optional with some reasonable default values